pavelib/assets.py

```diff
@@ -56,17 +70,21 @@ def configure_paths():
         lms_css = theme_dir / "lms" / "static" / "css"
         if lms_sass.isdir():
             lms_css.mkdir_p()
-            THEME_SASS_DIRECTORIES.append(lms_sass)
+            THEME_SASS_DIRECTORIES.append(("lms/static/sass", theme_dir / "lms" / "static" / "css"))
+            THEME_SASS_DIRECTORIES.append((lms_sass, lms_css))
+            THEME_SASS_LOOKUP_DIRECTORIES.append(lms_sass / "partials")
         cms_sass = theme_dir / "cms" / "static" / "sass"
         cms_css = theme_dir / "cms" / "static" / "css"
         if cms_sass.isdir():
             cms_css.mkdir_p()
-            THEME_SASS_DIRECTORIES.append(cms_sass)
+            THEME_SASS_DIRECTORIES.append(("cms/static/sass", theme_dir / "cms" / "static" / "css"))
```

Comment:
Nit: You can replace `theme_dir / "cms" / "static" / "css"` with `cms_css`, which is already defined a couple of lines above.